exercises/practice/palindrome-products/.meta/proof.ci.js

```diff
@@ -70,14 +68,22 @@ export class Palindromes {
   }
 
   get smallest() {
-    for (let m = this.minFactor; m <= this.maxFactor; m += 1) {
```

Comment:
_Isn't this the same exact algorithm just written in a much more complex manner?_  Isn't the loop still covering exactly the same territory?  Since you're just finishing the first item you can't get the same benefits you could by optimization the max loop.

Reply:
Please see #1407. I have updated the code after our discussion on exercism.io

Reply:
Can't this be resolved by simply having the second for loop start with `m` instead of `minFactor`, avoiding looping over the same territory as prior passes?

Reply:
Adding complexity solely for symmetry is not a win IMHO.  If the simpler for loop solution work for minimum, lets not complicate it.